.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    name: Run tests
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout code
+      - name: Stop postgres
+        run: |
+          sudo service postgresql stop
+      - name: Docker set up
+        run: |
+          docker run -e POSTGRES_PASSWORD=password -d --net host --name postgres $(docker build -q .travis/postgres) -c wal_level=logical -c max_replication_slots=1 -c max_wal_senders=1
```

Comment:
This is still referring to a file in the .travis directionry. It is not a dependency on travis. The file just happen to be there. The followup PR that removes the travis script will move that file as well.